goleveldb.go

```diff
@@ -24,7 +25,13 @@ type GoLevelDB struct {
 var _ DB = (*GoLevelDB)(nil)
 
 func NewGoLevelDB(name string, dir string) (*GoLevelDB, error) {
-	return NewGoLevelDBWithOpts(name, dir, nil)
+	options := &opt.Options{
+		Filter:                 filter.NewBloomFilter(10),
+		OpenFilesCacheCapacity: 8192,
+		BlockCacheCapacity:     opt.GiB,
```

Comment:
What was the default here?

Reply:
200 on mac, 500 on other.

Reply:
Can we add comments to say what the normal default was.

Reply:
absolutely. 

Sorry for delay on this, been real busy.